test/hotspot/jtreg/compiler/vectorapi/TestReverseByteTransforms.java

```diff
@@ -39,6 +39,7 @@
  * @bug 8287794
  * @summary Test various reverse bytes ideal transforms on X86(AVX2, AVX512) and AARCH64(NEON)
  * @requires vm.compiler2.enabled
+ * @requires !(vm.cpu.features ~= ".*sve.*")
```

Comment:
Please add a comment such as:
"On SVE system, we have a specific optimization, `ReverseBytesV (ReverseBytesV X MASK) MASK => X`, which eliminates both ReverseBytesV nodes. This optimization rule is specifically on hardware with native predicate support.
There is an SVE specific case TestReverseByteTransformsSVE.java, so this patch simply marks TestReverseByteTransforms.java as non-SVE only.